src/app/cli/src/coda_processes.ml

```diff
@@ -11,16 +11,25 @@ let init () = Parallel.init_master ()
 let net_configs n =
   let external_ports = List.init n ~f:(fun i -> 23000 + (i * 2)) in
   let discovery_ports = List.init n ~f:(fun i -> 23000 + 1 + (i * 2)) in
+  let ips =
+    List.init n ~f:(fun i ->
+        Unix.Inet_addr.of_string @@ sprintf "127.0.0.%i" (i + 10) )
```

Comment:
don't want to start at 1?

Reply:
I think it's better if we make it obvious we're doing something weird with the IPs.

Reply:
OK